src/components/pie-author/pie-author.tsx

```diff
@@ -357,16 +338,45 @@ export class Author {
     }
   }
 
+  getElementByType = (elements: {[key: string]: string;}, type: string) => {
+    for (const [key, value] of Object.entries(elements)) {
+      if (value.startsWith(type)) {
+        return key;
+      }
+    }
+
+    return null;
+  }
+
   getPieContentModelWithToggledComplexRubric = async ({complexRubricCheckedValues, pieContentModel}) => {
     const {
       shouldAddComplexRubric,
       shouldRemoveComplexRubric,
+      shouldForceEnableComplexRubric,
       complexRubricElements
     } = complexRubricCheckedValues || {};
 
     if (shouldAddComplexRubric) {
+      let clonedModel = cloneDeep(pieContentModel);
+
+      // if we are forced to convert the rubric, get the default values from it
+      if (shouldForceEnableComplexRubric) {
+        const rubricName = this.getElementByType(pieContentModel.elements, '@pie-element/rubric');
+        const rubric = pieContentModel.models.find((el) => el.element === rubricName);
+        if (rubric) {
+          const simpleRubric = _omit(rubric, ['id', 'element']);
+          this.defaultComplexRubricModel = {
+            rubrics: {
+              simpleRubric
+            }
+          };
+
+          clonedModel = this.removeRubricItemTypes(clonedModel, [rubric.id]);
```

Comment:
Would it be bad if we'd use an array of ids, in case there are more? Should be the case, but we're handling this as an array anyway inside removeRubricItemTypes